.changeset/fuzzy-colts-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': minor
```

Comment:
Think we could call this a patch tbh? Not really new features or APIs being added.